sign/eddilithium3/eddilithium_test.go

```diff
@@ -0,0 +1,106 @@
+package eddilithium3
+
+import (
+	"encoding/binary"
+	"testing"
+)
+
+func BenchmarkVerify(b *testing.B) {
+	// Note that the expansion of the matrix A is done at Unpacking/Keygen
+	// instead of at the moment of verification (as in the reference
+	// implementation.)
+	var seed [32]byte
+	var msg [8]byte
+	var sig [SignatureSize]byte
+	pk, sk := NewKeyFromSeed(&seed)
```

Comment:
Let's promote the use of `GenerateKey` instead of `NewKeyFromSeed`.

Reply:
I agree for code examples, but this is a benchmark.  Why use `GenerateKey` here?

Reply:
For the same reason, promoting the use of `GenerateKey`. In fact, I am not sure, if we need `NewKeyFromSeed`. Disclaimer: I added `NewKeyFromSeed` in Ed25519 to be compatible with stdlib, but I think the function `Public()` does the same job.

Reply:
In some situations you want to deterministically generate a key pair from a base secret.  For instance if you want to have one secret and generate a new key pair for each website/contact you have to prevent linking.

Reply:
> In some situations you want to deterministically generate a key pair from a base secret. For instance if you want to have one secret and generate a new key pair for each website/contact you have to prevent linking.

This is a use case that is not part of the algorithmic definition.

Reply:
I don't understand that?

Isn't what I describe is what is essentially used for U2F?

Reply:
I am not saying that `NewKeyFromSeed` is useles at all, am only pointing that l prefer to use GenerateKey.

Reply:
Yes, I agree to use `NewKeyFromSeed` in code examples, but this is a benchmark.

Reply:
all right, let's move forward as is.